NOTICE.md

```diff
@@ -1,6 +1,14 @@
 HASH: 152b6b858d2f6b94fa398fa4545d5e130ab24434
 #############################################################
-WARNING: This update bumps to rpi-4.19.y linux tree
+WARNING: 'rpi-update' updates to pre-releases of the linux 
+kernel tree and Videocore firmware.
+
+'rpi-update' should only be used if there is a specific 
+reason to do so. For example, a request by a Raspberry Pi 
+engineer.
```

Comment:
The new second sentence has no verb, whereas before the split the verb was "used".

Reply:
Sorry, cannot see the issue.

Reply:
In the following sentence, where is the verb?:
```
For example, a request by a Raspberry Pi Engineer.
```

Reply:
Does it need one? The 'for example' implicitly refers back to the previous sentence where there is a verb? Gets a bit cluncky if you have to say something like "For example, a request to use it by a raspberry pi engineer"?

Reply:
done